### ISSUE TITLE
Swift4.1.2 support

### ISSUE THREAD

**Issue by noppefoxwolf** [RxSwiftCommunity/RxKeyboard#66]
```
Incompatible Swift version - framework was built with 4.1 (swiftlang-902.0.48 clang-902.0.37.1) and the local version is 4.1.2 (swiftlang-902.0.54 clang-902.0.39.2).
```

**Comment by freak4pc** [comment]
This is the main issue with no ABI stability and the fact this specific repo uses ZIP Archives for releases. It means every major, minor and patch bump should get a new build. So I think we'll need to release 0.8.4 for this bump, probably. 

@devxoul is probably flying to WWDC so I imagine he won't have time to do this soon, but you may make a new binary build and a PR if you want for the time being

**Comment by bobgodwinx** [comment]
@noppefoxwolf Waiting for the official release of [RxSwift. Here is the blocker](https://github.com/ReactiveX/RxSwift/issues/1555) I have a PR with `Carthage` version of `RxSwift` but just wait a bit and it would be fixed.

**Comment by devxoul** [comment]
I released [0.8.3](https://github.com/RxSwiftCommunity/RxKeyboard/releases/tag/0.8.3) which support Swift 4.1.2 (swiftlang-902.0.54 clang-902.0.39.2).
